staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -45,20 +64,20 @@ type Transformer interface {
 	// TransformFromStorage may transform the provided data from its underlying storage representation or return an error.
 	// Stale is true if the object on disk is stale and a write to etcd should be issued, even if the contents of the object
 	// have not changed.
-	TransformFromStorage(data []byte, context Context) (out []byte, stale bool, err error)
+	TransformFromStorage(data []byte, context context.Context) (out []byte, stale bool, err error)
```

Comment:
I expected `TransformFromStorage(ctx context.Context, data []byte, dataContext Context)`, not plumbing the existing second parameter as a `context.Context` value. The colliding interface name is unfortunate, but if we were going to fix that, a rename of the param and/or the interface could work

Reply:
Yep, I realized after doing this that you wanted that change. Will do :)